.golangci.yml

```diff
@@ -4,8 +4,8 @@ run:
   build-tags:
     - examples
   skip-dirs:
-    - apiv6/localhost_apiserver/cloudexport
-    - apiv6/localhost_apiserver/synthetics
+    - localhost_api_fakeserver/cloudexport
```

Comment:
When we were discussing renaming localhost_apiserver, I meant just _fakeserver_, not _localhost_api_fakeserver_. _localhost_ is unimportant implementation detail and we have _api_ already in _kentikapi_ parent directory, so I think they are redundant.
Please rename to _fakeserver_